docs/source/getting-started/installation.rst

```diff
@@ -22,7 +22,9 @@ only have access to certain clouds, use any combination of
 :code:`"[aws,azure,gcp]"` (e.g., :code:`"[aws,gcp]"`) to reduce the
 dependencies installed.
 
-You may also install SkyPilot from source.
+SkyPilot also supports a fourth cloud provider, `Lambda <https://lambdalabs.com/>`_, if you install SkyPilot from source.
+
```

Comment:
We plan to cut a release soon - so perhaps we can remove these changes?

Reply:
Agreed

Reply:
Updated!